src/Posts.js

```diff
@@ -1,22 +1,37 @@
-import React, { Component } from "react";
+import React, { useState, useEffect } from 'react';
+import { myConfig } from './modules/config';
+const sanitizeHtml = require('sanitize-html');
 
-class Posts extends Component {
-    render() {
-        return (
-            <div>
-                <h2>POSTS</h2>
-                <p>Mauris sem velit, vehicula eget sodales vitae,
-                    rhoncus eget sapien:</p>
-                <ol>
-                    <li>Nulla pulvinar diam</li>
-                    <li>Facilisis bibendum</li>
-                    <li>Vestibulum vulputate</li>
-                    <li>Eget erat</li>
-                    <li>Id porttitor</li>
-                </ol>
-            </div>
+export default function Posts(props) {
+    const [posts, setPosts] = useState([
+        {
+            posts: []
+        }
+    ]);
+
+    async function fetchPostsData() {
+        const response = await fetch(
+            myConfig.apiUrl + myConfig.ptsEnd
```

Comment:
Mostly a nitpick, but in ES6 you should be able use backtick interpolation, which is generally a more legible way to concatenate strings. For example, my first thought here was did you miss a '/'? True, I would probably have asked that question with a backtick interpolation too, but it's less easy to make those sorts of mistakes with backtick interpolation.

Also, `ptsEnd`? Was this meant to be `postsEnd`? Is there a reason you're shortening it like this. If not, a name like `postsPath` would be more obvious.

Reply:
For the first one, it's because I didn't know backtick interpolation was a thing so I'll have to look at that sometime. As for `ptsEnd` I mean it as `postsEndpoint` / `postPath`. I got into the habit of shortening names like that from one of my Java lecturers in college. It's something I'm trying to break but it's an ingrained habit.